husky_components/tabs/main.js

```diff
@@ -50,6 +50,7 @@ define(function() {
             forceReload: false,
             callback: null,
             forceSelect: true,
+            conditionData: {},
```

Comment:
hmm also formData is not correct here. because every kind of data could be passed here. i think conditionData is not "nice" but it describes it quite good. they are data to check the conditions.